### ISSUE TITLE
Not connecting to the database

### ISSUE THREAD

**Issue by stratacast** [keepassxreboot/keepassxc-browser#1276]
## Expected Behavior

Opening a web page should show that the database is connected and that there are entries (if present for the URL)

## Current Behavior

There is no connection to the database and I have to click "Redetect Logins" and refresh the page to get the connection

## Steps to Reproduce (for bugs)

1. Open website with known entry
2. Look to see the database isn't connected
3. Click redetect logins
4. Refresh the webpage
5. The connection appears and I can fill in entries

## Debug info

KeePassXC - 2.6.4
KeePassXC-Browser - 1.7.6
Operating system: Intel Mac & FreeBSD
Browser: Firefox

**Comment by varjolintu** [comment]
What's the page URL? Or does this happen every time on every page?

**Comment by stratacast** [comment]
It has been happening a lot. 1Password (lol, work), GitHub, Amazon, my own Rocket Chat server. I'd say most of the time. Maybe sometimes there's a site where it will properly connect.

**Comment by varjolintu** [comment]
Can you record a small screencast to show how the problem displays for you?

**Comment by stratacast** [comment]
Of course! Before I do that though, is it possible at all that a ghetto internet connection could prevent the extension from working properly?

**Comment by varjolintu** [comment]
Connection speed should not matter. I also recommend trying a new temporary browser profile.

**Comment by stratacast** [comment]
The more I'm doing this the more I'm seeing that the sites that are having these issues have too much personally identifiable information. It seems like maybe the new profile works, but then when I switch back to my default I don't have the problem. It is rather sporadic. But to give more details:

* When the issue occurs the keepassxc icon is invisible in the form fields
* I can click it but it says no connection to database
* Same thing with the toolbar icon, it shows a little red X on the icon and says no connection to database
* Clicking redetect login fields a couple times and refreshing the page then allows me to use the extension as expected

I'll do a screencast next time this arises, and hope it doesn't show up on a site with too much private info. I was going to record 1Password's screen until I saw how much information it exposes at login

**Comment by varjolintu** [comment]
@stratacast When this happens, can you confirm keepassxc-proxy is still running? Sounds like it's not available anymore.

**Comment by stratacast** [comment]
Can confirm keepassxc-proxy is still running.

**Comment by polearnik** [comment]
confirm the bug. usually it happens then keepass and browser was open and  pc is go to standby mode  .
After wake up browser connection say "no connection" if keepass is opened
 I have enabled setting to block keepass if pc goes to standby mode

**Comment by polearnik** [comment]
@varjolintu  It can be fixed in some way?

**Comment by varjolintu** [comment]
@polearnik I'm not sure exactly. Needs to be reproduced and tested properly.

**Comment by Eleison23** [comment]
I am affected by this bug as well. KeepassXC 2.6.6 AppImage, Fedora 32 KDE, Chrome 92.0.4515.131 (Official Build) (64-bit)
I've ruled out standby.
For half the sites I go to, I get the red X, then I reload and reconnect and reload again and twiddle a few times and eventually it realizes I need it.
The browser integration was really cool a while ago, since I had been using finicky AutoType. Now they're both finicky! Welcome to District 4; May the Odds be Ever in your Favour!

**Comment by varjolintu** [comment]
> I am affected by this bug as well. KeepassXC 2.6.6 AppImage, Fedora 32 KDE, Chrome 92.0.4515.131 (Official Build) (64-bit)
> I've ruled out standby.
> For half the sites I go to, I get the red X, then I reload and reconnect and reload again and twiddle a few times and eventually it realizes I need it.
> The browser integration was really cool a while ago, since I had been using finicky AutoType. Now they're both finicky! Welcome to District 4; May the Odds be Ever in your Favour!

This is really strange, because there haven't been any major changes for a while for the browser integration. If downgrading KeePassXC doesn't help, then it must be a distro or Chrome issue. Which of course is not any better. Things should just work. Still, I'd recommend trying the downgrade and maybe another browser (Firefox). Any idea if this happened with a distro upgrade or something similar? I recall seeing some reports concerning new Fedora release.

**Comment by stherold** [comment]
For me every sizes login fields look like this after I restarted my mac and unlocked the database. The connection is still intact in the extension options but I have to re-connect (delete connected/ connect again)

![Screen Shot 2022-04-07 at 10 42 39](https://user-images.githubusercontent.com/64915710/162158628-a7b30bc3-6606-4017-9aee-1ef307a8cdb2.png)

**Comment by varjolintu** [comment]
@stherold You have to delete the whole connection? Or just "Reload" from the extension popup? Sounds like it's an issue with starting or re-enabling `keepassxc-proxy`. Restarting Chrome should restart it too. Can you confirm the `keepassxc-proxy` process is running when this happens?

**Comment by stherold** [comment]
Sorry my fault! I did not know this functionality from the popup. I just try to understand all the features of KeePassXC :D But still: Is the reload intendend?

**Comment by varjolintu** [comment]
@stherold When KeePassXC is started after the browser, it must be reconnected. Keeping KeePassXC open in the background in macOS usually solves these issues.

**Comment by Chealer** [comment]
@stratacast does this happen only after hibernation?
